Formula/corral.rb

```diff
@@ -4,6 +4,7 @@ class Corral < Formula
   url "https://github.com/ponylang/corral/archive/0.6.0.tar.gz"
   sha256 "d1e4cfd07c170780595b4681ad444faf69892d59adec4a51b02ede5641a4fdd2"
   license "BSD-2-Clause"
+  revision 1
```

Comment:
I don't think this is worth a `revision` bump. There are only 5 installs on Linux in the past year. See https://machomebrew.slack.com/archives/C06G173B7/p1664789129096879.

If we really want a rebuild, we can remove the Linux bottle separately then dispatch a new one.

Reply:
If we're going to do that, I'd like to do a CI run to confirm that actually fixes the problem.

Reply:
Could you be more specific about what you mean by a CI run? Building a new Linux bottle separately is a CI run, and also calls `brew test`, so I'm not sure what else you're looking to do, if any.

Reply:
Uh I guess they're the same thing actually.  For some reason I was thinking they needed be done together but they don't.  It is too bad we don't have way to say "only revision bump this on Linux" (or only on macOS, but that seems to be needed less often) because it doesn't make sense to revision bump formulae for both platforms when the fix is only needed for one.  I'm not yet sure how that would be implemented but it would be useful in cases like this.

On a somewhat related note:  could we use the "delete the bottle line and then dispatch a new one" trick for the cases where we need a new bottle on Linux because of the migration?  That avoids doing a revision bump only for Linux, especially when the bottle isn't even broken for users, just CI.  `lzlib` is currently affected by this, for example.